tools/rosparam/src/rosparam/__init__.py

```diff
@@ -150,6 +151,12 @@ def print_params(params, ns):
     
 # yaml processing
 
+def yaml_load(str):
+    return yaml.load(str, Loader=ROSSafeLoader)
+
+def yaml_load_all(str):
+    return yaml.load_all(str, Loader=ROSSafeLoader)
```

Comment:
The naming of both functions is misleading since it doesn't mention that it is actually using a safe loader.

Reply:
As mentioned below I will remove these functions and instead add the resolvers to the `SafeLoader` object in the `yaml` module.